src/assets/locales/fr.json

```diff
@@ -0,0 +1,465 @@
+{
+  "app-name": "Application",
+  "editor-name": "Editeur de scène",
+  "contact-email": "app@company.com",
+  "sign-in.prompt": "Identifiez-vous pour épingler des objets dans les salles.",
+  "sign-in.admin": "Vérifiez votre adresse pour la vérification par e-mail. Une fois vérifié, entrez votre e-mail pour créer votre compte ou bien identifiez-vous.",
+  "sign-in.admin-no-permission": "Vous n'avez pas le droit d'accéder aux outils d'aministration. Identifiez-vous avec un autre compte ou demandez l'accès à un administrateur.",
+  "sign-in.hub": "Un compte est nécessaire pour rejoindre une salle.\n\nEntrez votre e-mail pour créer votre compte ou bien identifiez-vous.",
+  "sign-in.auth-started": "E-mail envoyé à {email} !\n\nPour continuer, ouvrez l'e-mail reçu sur votre téléphone, tablette ou ordinateur et cliquez sur le lien.\n\nVous n'avez pas reçu d'e-mail ? Vous ne pourrez pas créer de compte.",
+  "sign-in.pin": "Vous devez vous identifier pour épingler des objets.",
+  "sign-in.pin-complete": "Vous êtes identifié.",
```

Comment:
I am not very fond of stuff such as "Vous êtes identifié-e", but I don't want people to say Hubs is not inclusive. What about "Identification complétée"?

I'm not even sure when this message pops out 🤔